crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -1212,6 +1214,7 @@ async fn mapper_publishes_supported_operations_for_child_device() {
     let (_tmp_dir, sm_mapper) = start_c8y_mapper(broker.port, &cfg_dir).await.unwrap();
 
     publish_a_fake_jwt_token(broker).await;
+    broker.publish("c8y/s/ds", "106,child-one").await.unwrap();
```

Comment:
Same question as the one above.

Reply:
@albinsuresh , yes because now we don't send 114 on startup but only when we receive a 116 from the cloud. The logic is the following:

mapper sends 105
cloud replies with 106
mapper computes diff between cloud and local
mapper sends 101 for new child devices
mapper sends 114 for all child devices